docs/UserGuide.md

```diff
@@ -216,6 +226,34 @@ Format: `scheme INDEXES`
 Examples:
 * `scheme 1` shows scheme that the 1st person in the address book is eligible for.
 
+### Archiving person : `archive`
+
+Archives the specified person from the address book.
+
+Format: `archive INDEX`
+
+* Archives the person at the specified `INDEX`.
+* The index refer to the index number shown in the displayed person list.
+* The index **must be a positive integer** not exceeding the last index number e.g. 1, 2, 3, …
+* The specified person must currently not be archived.
+
+Examples:
+* `list` followed by `archive 2` archives the 2nd person in the address book.
+
+### Unarchiving person : `unarchive`
+
+Unarchives the specified person from the address book.
+
+Format: `unarchive INDEX`
+
+* Archives the person at the specified `INDEX`.
```

Comment:
Seems like you may have mistakenly put "archives" instead of "unarchives"?

Reply:
Yes thanks for spotting out this typo